web/components/Footer/Footer.tsx

```diff
@@ -62,26 +70,15 @@ export const Footer = () => (
 
     <GridWrapper>
       <ul className={styles.links}>
-        <li className={styles.linksItem}>
-          <Link href="/code-of-conduct">
-            <a className={styles.linksItemLink}>Code of conduct</a>
-          </Link>
-        </li>
-        <li className={styles.linksItem}>
-          <Link href="/privacy-policy">
-            <a className={styles.linksItemLink}>Privacy policy</a>
-          </Link>
-        </li>
-        <li className={styles.linksItem}>
-          <Link href="/contact">
-            <a className={styles.linksItemLink}>Contact</a>
-          </Link>
-        </li>
-        <li className={styles.linksItem}>
-          <Link href="/venues">
-            <a className={styles.linksItemLink}>Venues</a>
-          </Link>
-        </li>
+        {links
+          ?.filter(({ slug }) => slug.current)
+          .map(({ name, slug }) => (
+          <li key={name} className={styles.linksItem}>
```

Comment:
Nit: Wonder if the key could be an `_id` or something instead… though duplicate names are pretty far-fetched in this case :)

Reply:
Fixed in 46cc540.